.appveyor.yml

```
@@ -61,7 +61,7 @@ init:
 # Required software for building
 install:
   # Update environment variables
-  - ps: $env:RZ_VERSION = ( python sys\\version.py )
+  - ps: $env:RZ_VERSION = (& (Join-Path $env:PYTHON python.exe) sys\\version.py )
```

```suggestion
  - ps: $env:RZ_VERSION = ( %PYTHON%\python.exe sys\\version.py )
```

The above is simpler if it works, and it's of the same form as line 68 below.

---

Oh it's PowerShell ... sorry